scripts/containers/App.js

```diff
@@ -0,0 +1,18 @@
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+
+import App from "../components/App";
+
+
+function mapStateToProps(state) {
+  return {};
```

Comment:
That's where you could map some store state properties to the child App component.